### PR TITLE
(REATTEMPT): Add pg_to_sql_qry and fix f-strings

### PR DESCRIPTION
**Pull request by cindyatnycdot**

### CODE REVIEW

File: pysqldb3/tests/test_io.py [hunk 1]
```diff
@@ -221,53 +221,200 @@
...
+            create table {pg_schema}.{test_pg_to_sql_qry_spatial_table}
+                (test_col1 int, test_col2 int, test_geom geometry);
+            insert into {pg_schema}.{test_pg_to_sql_qry_spatial_table} (test_col1, test_col2, test_geom)
+                 VALUES (1, 2, ST_SetSRID(ST_MAKEPOINT(-71.1043443253471, 42.3150676015829), 2236));
+            insert into {pg_schema}.{test_pg_to_sql_qry_spatial_table} (test_col1, test_col2, test_geom)
+                VALUES (3, 4, ST_SetSRID(ST_MAKEPOINT(91.2763, 11.9434), 2236));
+        """)
+
+        data_io.pg_to_sql_qry(db, sql, query=f"""
+                                               SELECT test_col1, test_col2, ST_AsEWKT(test_geom) test_geom --comments within the query
```
Comment (shostetter) [inline pysqldb3/tests/test_io.py:375]: you should transfer it as regular geom but compare the results using wkt. Something like this 


data_io.pg_to_sql_qry(db, sql, query=f"""
                                               SELECT test_col1, test_col2, test_geom --comments within the query
                                                FROM {pg_schema}.{test_pg_to_sql_qry_spatial_table} -- geom here
                                                -- end here""",
                              dest_table= test_pg_to_sql_qry_spatial_table,
                              dest_schema = sql_schema,                      
                              spatial = True
                              )

pg_df = db.dfquery(f"""
        select test_col1, test_col2,  ST_Astext(est_geom) test_geom
        from {pg_schema}.{test_pg_to_sql_qry_spatial_table}
        order by test_col1
        """)
sql_df = sql.dfquery(f"""
                select test_col1, test_col2, test_geom.STAsText() as test_geom
                from {sql_schema}.{test_pg_to_sql_qry_spatial_table}
                order by test_col1
        """)

**Review by shostetter — COMMENTED** [review]
I left 1 comment about the spatial query test

**Comment by cindyatnycdot** [comment]
I fixed the spatial test for `pg_to_sql_qry` but I cannot figure out how to successfully move a geometry from sql to pg (the other way around). The geom doesn't exist in the pg table.

**Comment by shostetter** [comment]
> I fixed the spatial test for `pg_to_sql_qry` but I cannot figure out how to successfully move a geometry from sql to pg (the other way around). The geom doesn't exist in the pg table.

this testing is working fine for me

**Review by shostetter — APPROVED** [review]

**Comment by cindyatnycdot** [comment]
> > I fixed the spatial test for `pg_to_sql_qry` but I cannot figure out how to successfully move a geometry from sql to pg (the other way around). The geom doesn't exist in the pg table.
> 
> this testing is working fine for me

Yes, it's passing only because i've written the test as `data_io.sql_to_pg_qry(sql, db, query=f""" sql_to_pg_qry("""SELECT test_col1, test_col2, CAST(test_geom.STAsText() AS VARCHAR) test_geom --comments within the query""", ....)`. i am not actually bringing a spatial table over, because the `test_geom` column is actually text right now. The geom does not get brought over to pg.

**Comment by shostetter** [comment]
> > > I fixed the spatial test for `pg_to_sql_qry` but I cannot figure out how to successfully move a geometry from sql to pg (the other way around). The geom doesn't exist in the pg table.
> > 
> > 
> > this testing is working fine for me
> 
> Yes, it's passing only because i've written the test as `data_io.sql_to_pg_qry(sql, db, query=f""" sql_to_pg_qry("""SELECT test_col1, test_col2, CAST(test_geom.STAsText() AS VARCHAR) test_geom --comments within the query""", ....)`. i am not actually bringing a spatial table over, because the `test_geom` column is actually text right now. The geom does not get brought over to pg (the ogr2ogr command doesn't work to do that. I don't know why)

Im confused, are you adding pg_to_sql_qry why are you adding tests for sql_to_pg_qry?

**Pull request merged**